tests/integration/wordpress_client.py

```diff
@@ -184,3 +186,32 @@ def list_plugins(self):
             except_status_code=200,
         )
         return [p["plugin"].split("/")[0] for p in response.json()]
+
+    def upload_media(
+            self, filename: str, content: bytes, mimetype: str = None
+    ) -> typing.List[str]:
+        """Upload a media file (image/video)
+
+        Return url of different sized for the uploaded file on WordPress.
```

Comment:
How about "Return URL of resized media file after upload to WordPress."? ("url" should be capitalised here, and "different sized" is a little unclear - feel free to reword as you see fit).

Reply:
Updated to `Return URL of the original image and resized images for the uploaded file on WordPress.`

WordPress will create a URL for the original image and a series of resized images.